jobs/future-effective-filings/config.py

```diff
@@ -28,119 +47,82 @@
 # this will load all the envars from a .env file located in the project root (api)
 load_dotenv(find_dotenv())
 
-CONFIGURATION = {
-    'development': 'config.DevConfig',
-    'testing': 'config.TestConfig',
-    'production': 'config.ProdConfig',
-    'default': 'config.ProdConfig'
-}
-
-
-def get_named_config(config_name: str = 'production'):
-    """Return the configuration object based on the name.
-
-    :raise: KeyError: if an unknown configuration is requested
-    """
-    if config_name in['production', 'staging', 'default']:
-        config = ProdConfig()
-    elif config_name == 'testing':
-        config = TestConfig()
-    elif config_name == 'development':
-        config = DevConfig()
-    else:
-        raise KeyError(f"Unknown configuration '{config_name}'")
-    return config
-
-
-class _Config(object):  # pylint: disable=too-few-public-methods
+class Config(object):  # pylint: disable=too-few-public-methods
     """Base class configuration that should set reasonable defaults for all the other configurations."""
 
-    NATS_CONNECTION_OPTIONS = {
-        'servers': os.getenv('NATS_SERVERS', 'nats://127.0.0.1:4222').split(','),
-        'name': os.getenv('NATS_CLIENT_NAME', 'entity.filing.worker')
-    }
-    STAN_CONNECTION_OPTIONS = {
-        'cluster_id': os.getenv('NATS_CLUSTER_ID', 'test-cluster'),
-        'client_id': str(random.SystemRandom().getrandbits(0x58)),
-        'ping_interval': 1,
-        'ping_max_out': 5
-    }
-
-    SUBSCRIPTION_OPTIONS = {
-        'subject': os.getenv('NATS_SUBJECT', 'entity.filings'),
-        'queue': os.getenv('NATS_QUEUE', 'filing-worker'),
-        'durable_name': os.getenv('NATS_QUEUE', 'filing-worker') + '_durable'
-    }
-
-    FILER_PUBLISH_OPTIONS = {
-        'subject': os.getenv('NATS_FILER_SUBJECT', 'entity.filing.filer')
-    }
-
     PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
 
-    COLIN_URL = os.getenv('COLIN_URL', '')
-    LEGAL_URL = os.getenv('LEGAL_URL', '')
-    AUTH_URL = os.getenv('AUTH_URL', '')
-    USERNAME = os.getenv('AUTH_USERNAME', '')
-    PASSWORD = os.getenv('AUTH_PASSWORD', '')
-    SENTRY_DSN = os.getenv('SENTRY_DSN', '')
+    ENVIRONMENT = os.getenv("ENVIRONMENT", "prod")
+    GCP_AUTH_KEY = os.getenv("GCP_AUTH_KEY", None)
+    ENTITY_MAILER_TOPIC = os.getenv("ENTITY_MAILER_TOPIC", "mailer")
```

Comment:
don't think we need the emailer topic for this job.  Can you confirm we only publish Q messages to the filer topic?

Reply:
True, makes sense. I'll be removing this line.

What do you mean by your question Argus?

Reply:
I was just wanting to confirm that this job only publishes filer queue messages.  If so, we can just remove all references in the job to the emailer Q topic.